examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -238,6 +238,7 @@ void PairingCommand::OnDiscoveredDevice(const chip::Dnssd::DiscoveredNodeData &
     ChipLogProgress(chipTool, "Discovered Device: %s:%u", buf, port);
 
     // Stop Mdns discovery. Is it the right method ?
+    CurrentCommissioner().StopCommissionableDiscovery();
```

Comment:
Presuming this comment could go away:

```suggestion
    // Stop Mdns discovery. Is it the right method ?
    CurrentCommissioner().StopCommissionableDiscovery();
```

Reply:
Yes, it probably can.